pymc3/distributions/bound.py

```diff
@@ -151,8 +151,8 @@ def __init__(self, distribution, lower, upper,
             default = None
 
         super().__init__(
-            distribution=distribution, lower=lower, upper=upper,
-            transform=transform, default=default, *args, **kwargs)
+            distribution, lower, upper,
```

Comment:
Don't we need the same change for `_DiscreteBounded` (and a test)?

Reply:
@twiecki done. Thanks!